plugins/scanners/fossid/src/main/kotlin/FossId.kt

```diff
@@ -249,7 +249,7 @@ class FossId internal constructor(
 
                 else -> {
                     val errorMessage = "Could not get project '$projectCode' for user '${config.user.value}: $error'"
-                    logger.error(errorMessage)
+                    logger.error { errorMessage }
```

Comment:
Should the errorMessage variable also go into the lambda?

Reply:
No, as it's also used for the exception one line below.